WHITEPAPER.md

```diff
@@ -151,18 +175,89 @@ The following phases may or may not be useful. I think it would be valuable if w
 
 #### Degrade Network Conditions
 
+Network is one of the greatest complexity, as well as fragility, facet of any
+systems. One often hears that network cannot be trusted.
+
+It is therefore a place of choice for any Chaos Engineer to look for weaknesses.
+The purpose is to gauge how one service copes with poor communication with
+a service it depends on.
+
+Common experiments can therefore:
+
+* add latency to a network call (either on the receive, send or both)
+* add jittering to create noise in the message
+* lose any number of packets along the channel
+* prevent name discovery
+* randomly close connections
+* inject random data into streams
+* send all data to nowhere
+
+Basically, any number of unhappy scenarios you can think of during a network
+exchange is a candidate.
+
 #### Vary Computing Resources
 
+Resources allocated to a service should not be trusted to always be available.
+Whether it's fighting for computing resources or attached devices going away,
+it is critical that you harness the impact of any of those situations on your
+system.
+
+* Reduce the available amount of CPU or memory to a service
+* Detach a device
```

Comment:
I have little bit of trouble understanding what it means concretely to "detach a device".

Reply:
Sorry for not being clear. I mean, remove a physical or virtual storage device for instance.

Reply:
Oh that makes sense now. Thank you